app/src/main/java/com/tip/lunchbox/model/server/response/Tokens.java

```diff
@@ -1,19 +1,17 @@
 package com.tip.lunchbox.model.server.response;
 
-import com.google.gson.annotations.Expose;
-
+// Required for Moshi
+@SuppressWarnings("unused")
 //Response for both signup and login endpoints
 public class Tokens {
-    @Expose
-    private String authToken;
-    @Expose
-    private String refreshToken;
+    private String auth_token;
```

Comment:
You are not following the camel case style of writing and because of this build tests are failing. Please fix this